app/upgrades/v19_2/upgrades.go

```diff
@@ -43,31 +40,14 @@ func AddCodeToERC20Extensions(
 	evmKeeper *evmkeeper.Keeper,
 ) (err error) {
 	logger.Info("Adding code to erc20 extensions...")
-	var (
-		// bytecode and codeHash is the same for all IBC coins
-		// cause they're all using the same contract
-		bytecode = common.FromHex(erc20Bytecode)
-		codeHash = crypto.Keccak256(bytecode)
-	)
 
 	erc20Keeper.IterateTokenPairs(ctx, func(tokenPair erc20types.TokenPair) bool {
 		// Only need to add code to the IBC coins.
 		if tokenPair.ContractOwner != erc20types.OWNER_MODULE {
 			return false
 		}
```

Comment:
please use proper method here:

```suggestion
		if !tokenPair.IsNativeCoin() {
			return false
		}
```